lib/chef/provisioning/transport/winrm.rb

```diff
@@ -114,9 +99,15 @@ def make_url_available_to_remote(local_url)
 
       def session
         @session ||= begin
-          require 'kconv' # Really, people? *sigh*
           require 'winrm'
-          ::WinRM::WinRMWebService.new(endpoint, type, options)
+          ::WinRM::Connection.new(options).shell(:powershell)
```

Comment:
Setting shell to powershell here may get rid of the need for the :raw block above